testing/testcafe/tests/scheduler/rerenderOnResize.ts

```diff
@@ -0,0 +1,66 @@
+import { getStyleAttribute, setStyleAttribute } from '../../helpers/domUtils';
+import createWidget from '../../helpers/createWidget';
+import url from '../../helpers/getPageUrl';
+import Scheduler from '../../model/scheduler';
+import { safeSizeTest } from '../../helpers/safeSizeTest';
+
+fixture.disablePageReloads`Re-render on resize`
+  .page(url(__dirname, '../container.html'));
+
+const createScheduler = async (container, options?): Promise<void> => {
+  await createWidget('dxScheduler', {
```

Comment:
Add return keyword
```suggestion
const createScheduler = async (container, options?): Promise<void> => {
  return await createWidget('dxScheduler', {
```

or remove brackets from the arrow function

```suggestion
const createScheduler = async (container, options?): Promise<void> => await createWidget('dxScheduler', {
```